### PR TITLE
Display ACL id for the private gateway

### PR DESCRIPTION
**Pull request by ravening**
### Description
Display link for acl id which is associated
with the private gateway

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [X] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [X] Trivial

### Screenshots (if appropriate):

Screenshot 2021-09-16 at 13 05 46

### CODE REVIEW

File: ui/src/components/view/InfoCard.vue [hunk 1]
```diff
@@ -414,6 +414,18 @@
...
         </div>
+
+        <div class="resource-detail-item" v-if="resource.aclid">
+          <div class="resource-detail-item__label">{{ $t('label.acl.id') }}</div>
+          <div class="resource-detail-item__details">
+            <span v-if="images.acl">
+              <resource-icon :image="getImage(images.acl)" size="1x" style="margin-right: 5px"/>
+            </span>
+            <a-icon v-else type="deployment-unit" />
+            <router-link :to="{ path: '/acllist/' + resource.aclid }">{{ resource.aclid }}</router-link>
```
Comment (rhtyd) [inline ui/src/components/view/InfoCard.vue:425]: Do ACL list have names, can we use that?

**Comment by nvazquez** [comment]
@blueorangutan ui

**Comment by rhtyd** [comment]
@davidjumani can you check or do you know if instead of aclid, acl name could be used?

File: ui/src/components/view/InfoCard.vue (hunk 1 above)
Comment (davidjumani) [inline ui/src/components/view/InfoCard.vue:425]:
```suggestion
            <router-link :to="{ path: '/acllist/' + resource.aclid }">{{ resource.aclname }}</router-link>
```

**Comment by davidjumani** [comment]
> @davidjumani can you check or do you know if instead of aclid, acl name could be used?

Yup it should work

**Comment by rhtyd** [comment]
@blueorangutan ui

**Comment by rhtyd** [comment]
@blueorangutan ui

**Comment by rhtyd** [comment]
@blueorangutan ui

File: ui/src/components/view/InfoCard.vue [hunk 2]
```diff
@@ -414,6 +414,18 @@
             <router-link :to="{ path: '/vpc/' + resource.vpcid }">{{ resource.vpcname || resource.vpcid }}</router-link>
           </div>
         </div>
+
+        <div class="resource-detail-item" v-if="resource.aclid">
+          <div class="resource-detail-item__label">{{ $t('label.acl') }}</div>
```
Comment (weizhouapache) [inline ui/src/components/view/InfoCard.vue:419]: no label.acl, please label.aclid or label.aclname instead.

```
ui/public/locales/en.json:"label.aclid": "ACL",
ui/public/locales/en.json:"label.aclname": "ACL Name",
```

**Comment by rhtyd** [comment]
@blueorangutan ui

**Review by rhtyd — APPROVED** [review]

**Review by weizhouapache — APPROVED** [review]
manually tested ok

**Review by utchoang — APPROVED** [review]

**Review by davidjumani — APPROVED** [review]

**Pull request merged**